erpnext/assets/doctype/asset_capitalization/asset_capitalization.py

```diff
@@ -77,15 +77,17 @@ def on_cancel(self):
 			"Stock Ledger Entry",
 			"Repost Item Valuation",
 			"Asset",
+			"Asset Movement"
```

Comment:
Theres a comma missing here, as indicated by the linter failure on this PR. After merging without fixing this, the linter fails for all other PRs 😢 .